enterprise_gateway/services/processproxies/k8s.py

```diff
@@ -10,10 +10,15 @@
 from typing import Any
 
 import urllib3
-from kubernetes import client, config
+import yaml
+from kubernetes import client
+from kubernetes.utils.create_from_yaml import create_from_yaml_single_item
+
+from enterprise_gateway.services.processproxies.k8s_client import kubernetes_client
 
 from ..kernels.remotemanager import RemoteKernelManager
 from ..sessions.kernelsessionmanager import KernelSessionManager
+from ..utils.envutils import is_env_true
```

Comment:
Could we refactor this filename to include an underscore for word separation?
```suggestion
from ..utils.env_utils import is_env_true
```